src/scripts.js

```diff
@@ -24,11 +29,17 @@ let searchInput = document.querySelector("#search-input");
 let showPantryRecipes = document.querySelector(".show-pantry-recipes-btn");
 let tagList = document.querySelector(".tag-list");
 let user;
+let users;
```

Comment:
Would love to change this variable name eventually so we don't have such similar variables. Maybe allUsers and currentUser. That's down the road though when we have time to refactor